CDP4CrossViewEditor.Tests/Assemblers/CrossviewPMeanArrayAssemblerTestFixture.cs

```diff
@@ -0,0 +1,701 @@
+            elementDefinition.Parameter.Add(this.pOnParameter);
+            elementDefinition.Parameter.Add(this.pStandByParameter);
+            elementDefinition.Parameter.Add(this.pDutyCycleParameter);
+            elementDefinition.Parameter.Add(this.pMeanParameter);
+
+            this.iteration.Element.Add(elementDefinition);
+
+            this.siteReferenceDataLibrary.ParameterType.Add(P_redundancy);
+            this.siteReferenceDataLibrary.ParameterType.Add(P_on);
+            this.siteReferenceDataLibrary.ParameterType.Add(P_stby);
+            this.siteReferenceDataLibrary.ParameterType.Add(P_duty_cyc);
+            this.siteReferenceDataLibrary.ParameterType.Add(P_mean);
+
+            this.session.Setup(x => x.OpenIterations).Returns(new Dictionary<Iteration, Tuple<DomainOfExpertise, Participant>>
+            {
+                {this.iteration, new Tuple<DomainOfExpertise, Participant>(this.domain, null)}
+            });
+
+            this.session.Setup(x => x.QuerySelectedDomainOfExpertise(It.IsAny<Iteration>())).Returns(this.domain);
+        }
+
+        [Test]
+        public void VerifyThatAssemblerCalculatesPMeanIfPDutyCycleAndPMeanAreProperlySet()
+        {
+            var crossviewSheetRowAssembler = new CrossviewSheetRowAssembler();
+            crossviewSheetRowAssembler.Assemble(this.elementDefinitions);
+            this.excelRows.AddRange(crossviewSheetRowAssembler.ExcelRows);
+
+            var arrayAssembler = new CrossviewArrayAssembler(this.excelRows, this.parameterTypes);
+
+            // The that power related parameters are present into the header
+            Assert.IsTrue(arrayAssembler.headerDictionary.ContainsKey("redundancy"));
+            Assert.IsTrue(arrayAssembler.headerDictionary.ContainsKey("P_on"));
+            Assert.IsTrue(arrayAssembler.headerDictionary.ContainsKey("P_stby"));
+            Assert.IsTrue(arrayAssembler.headerDictionary.ContainsKey("P_duty_cyc"));
+            Assert.IsTrue(arrayAssembler.headerDictionary.ContainsKey("P_mean"));
+
+            Assert.AreEqual(@"Crossview_ED_1.P_duty_cyc\PS_1", arrayAssembler.NamesArray[6, 11]);
+            Assert.AreEqual(@"Crossview_ED_1.P_duty_cyc\PS_1", arrayAssembler.NamesArray[6, 12]);
+            Assert.AreEqual(@"Crossview_ED_1.P_mean\PS_1", arrayAssembler.NamesArray[6, 13]);
+            Assert.AreEqual(@"Crossview_ED_1.P_mean\PS_1", arrayAssembler.NamesArray[6, 14]);
+        }
+
+        [Test]
+        public void VerifyThatAssemblerCalculatesPMeanIfRedundancyTypeIsInternal()
+        {
+            var crossviewSheetRowAssembler = new CrossviewSheetRowAssembler();
+            crossviewSheetRowAssembler.Assemble(this.elementDefinitions);
+            this.excelRows.AddRange(crossviewSheetRowAssembler.ExcelRows);
+
+            var arrayAssembler = new CrossviewArrayAssembler(this.excelRows, this.parameterTypes);
+
+            // The that power related parameters are present into the header
```

Comment:
Typo in comment "the that power related..."?

Given that this is repeated to many times, I would just remove this comment completely.